tests/functionnal/Ticket.php

```diff
@@ -4509,4 +4509,69 @@ public function testNeedReopen()
         $this->integer($ticket->fields['status'])->isEqualTo(\Ticket::ASSIGNED);
         $this->boolean((bool)$ticket->needReopen())->isFalse();
     }
+
+    protected function prepareInputForUpdateHandleActorChangesProvider()
+    {
+        return [
+            [
+                'input' => [
+                    '_additional_requesters' => [
+                        'users_id' => 1
+                    ],
+                    '_users_id_requester' => 2,
+                    '_additional_observers' => [
+                        'users_id'  => 1
+                    ],
+                    '_users_id_observer'  => 3,
+                    '_users_id_assign'    => 4,
+                    '_additional_groups_assigns' => [1],
+                    '_groups_id_assign'      => [2],
+                    '_suppliers_id_assign'      => [4],
+                ],
+                'changes' => [
+                    '_additional_requesters' => [ // Does not make sense, but whatever. It works I guess...
```

Comment:
Makes more sense now.
```suggestion
                    '_additional_requesters' => [
```